### ISSUE TITLE
Dreamng out loud: An Evo equivalent theme for WordPress

### ISSUE THREAD

**Issue by nick0** [modxcms/evolution#763]
Hello,

I have been building more WordPress sites of late (premium themed) due to client budget / time constraints and/or client requests due to WordPress's popularity but the problem I see is that WordPress themes are generally pretty clunky and much less flexible than building a site in Evo.

Are there any Evo (or Revo users out there) who might be interested in helping develop a MODx inspired WordPress theme with in built MODx awesomness that does not rely on third party plugins for basic stuff?

There are so many things that Evo can handle natively very well but which WP requires plugins for.

I am thinking something with the MODx equivalents of:
- chunks (useable via WP shortcodes)
- template variables (useable via WP shortcodes)
- snippets (including an IF or PHx equivalent, copyright snippet) (useable via WP shortcodes)
- modules (may not be needed)
- template editing (very important as this is clearly lacking in WP. the status quo is per page editing but this leads to inefficiencies at times)
- \- the ability to add a section to the template that only shows content on a page if it exists in the "template variable field" for that page would be nice
- \- Edit the page to edit the section. No more going to numerous sections of the WP Admin
- \- - If empty, show nothing
- \- - If has content, show the section
- manager manager equivalent to streamline the WP interface per user
- customiseable / templateable maintenance mode (easy inbuilt online/offline)
- MODx like placeholders that can be used through templates and pages (eg blog name, page title, description etc to make rolling out new sites easier)
- ability to set WYSIWYG visual / text (rich text / plain text) per each page
- ability to create/edit custom login URL straight out of the box (not requiring on yet another plugin)
- ability to choose/edit custom WP table prefix during setup
- a kick ass templateable menu builder like Ditto and/or WayFinder that can work with text and images
- separating style from content, something that Evo handles brilliantly (too often there are inconsistencies in WP themed sites due to the use of infinite styles in different spots)

Imagine a WP theme that is easily templateable and which mimics all the great things about Evo / Revo but which leverages the good things about WordPress like:
- user reach (it powers what 21%+ of all websites) 
- the official ongoing support of the WordPress system
- user management
- blogging
- commenting
- ability to use good third party plugins
  etc

Basically something that is intuitive, streamlined (meaning non bloated) and which allows easy template editing. Something that works as nicely as Evo (or Revo) on the WordPress platform on shared hosting without the reliance for too many third party plugins to handle basic things.

If we get it right we could even release it on Theme Forest or similar.

I am not a developer but am thorough at testing and handy with ideas, branding etc.

Anyone interested in exploring this with me? I would love to hear your thoughts

**Comment by dmi3yy** [comment]
For MODX, you can create themes quite easily, but for some reason, very few people do it. Soon we plan to improve the work with themes in EVO after which it will be easier to create them
